tcms/kiwi_auth/views.py

```diff
@@ -37,7 +37,7 @@ class Register(View):
     form_class = RegistrationForm
     success_url = reverse_lazy('core-views-index')
 
-    def post(self, request, *args, **kwargs):
+    def post(self, request):
```

Comment:
Bring these back b/c the signature of these methods contain args and kwargs. pylint-django isn't smart enough for the moment to understand this.